redis-om-spring/src/main/java/com/redis/om/spring/RedisOMSpringProperties.java

```diff
@@ -0,0 +1,18 @@
+package com.redis.om.spring;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(
+        prefix = "redis.om.spring"
+)
+public class RedisOMSpringProperties {
+    private int maxQueryLimit = 10;
```

Comment:
Let's make this 10,000 which I think is more realistic than 10. If they want to go higher then they'll need to add configuration.

Reply:
Setting this to 10 was just to avoid breaking changes. If you are fine with those changes, I'll gladly update it.

Reply:
We're still pre 1.0.0 so break away! :-)